script.js

```diff
@@ -330,4 +330,35 @@ function formatPrice(price) {
     var pSplit = p.split('.');
     pSplit[0] = pSplit[0].replace(/\B(?=(\d{3})+(?!\d))/g, ',');
     return t.replace('1', pSplit.join('.'));
+}
+
+function parsePriceByCountry(priceStr, currency) {
+    let site = window.location.host;
+    site = site.toLowerCase();
+    currency = currency.toUpperCase();
+
+    // default for: | .com | .de | .co.uk | co.jp | 
+    let thousandsSeperator = ',';
+    let fractionSeperator = '.';
+
+    // for: | .it | .es | .fr
+    if (site.endsWith('.it') || site.endsWith('.es') || site.endsWith('.fr')) {
+        // avoid for ils in checkout page
+        if (currency != 'ILS') {
```

Comment:
Why we need a special condition for ILS? don't we have this issue in other currencies?
Merging anyway, but we need to check it.
Thanks :)

Reply:
this condition solves the following problem:

![image](https://user-images.githubusercontent.com/1807493/107042033-f0255d00-67c9-11eb-8604-5b0ce365c4fe.png)


the host is amazon.it,  but the currency is ILS
I'm sure theres a smarter way to map all behaviour, but lets start with something... 
I'm still learning all the different possibilites